src/mlpack/methods/lsh/lsh_search_impl.hpp

```diff
@@ -554,9 +554,9 @@ void LSHSearch<SortPolicy, MatType>::GetAdditionalProbingBins(
   // Actions vector describes what perturbation (-1/+1) corresponds to a score.
   arma::Col<short int> actions(2 * numProj); // will be [-1 ... 1 ...]
   actions.rows(0, numProj - 1) = // First numProj rows.
-    -1 * arma::ones< arma::Col<short int> > (numProj); // -1s
+    -1 * ones< arma::Col<short int> > (numProj); // -1s
```

Comment:
```suggestion
    -1 * ones<arma::Col<short int>> (numProj); // -1s
```
This was written so long ago that you couldn't put two closing `>>`s next to each other :smile: